### ISSUE TITLE
No highlights displayed in Chrome (after import from G. Bookmarks)

### ISSUE THREAD

**Issue by peter-kehl** [ldenoue/yawas#14]
Successfully imported to Chrome bookmarks (Yawas settings: chrome.storage.sync (max 512 entries); chrome://settings/syncSetup/advanced > Customise sync > turned off sync of Bookmarks): "Import completed: 3620 Google Bookmarks are now in your Chrome Bookmarks!". Bookmarks (whether with highlights or not) seem to show up in chrome://bookmarks/?id=1020 (which shows Other bookmarks > Yawas) - I can't easily confirm if it's all of them. Imported highlights seem to be in the title field - right of `#_#`.

However, when I open those (bookmarked/target) URLs in same Chrome, Yawas doesn't seem to highlight the text that is right of `#_#` and between `<<..>>`. The same highlights continue to work with the existing (May 2021) version 7.3.3 of Yawas for Firefox.

Personally, I'm only interested in Firefox version - and I believe that Firefox is gaining more momentum than Chrome, so this is only an FYI.

I'd be happy to send you my GoogleBookmarks.html export file, or assist in any other way.

**Comment by ldenoue** [comment]
@peter-kehl correct: because the Chrome Bookmarks API only gives us `title` and `url` to store data, and because there doesn't seem to be a limit to the length of the `title` field, I chose to use it to store the original title + '#__#' + your highlights.
Could you send me the url and title (with highlights) of the bookmark that causes issues?
